File: app/src/main/java/com/cashfree/sdk_sample/java/MainActivity.java
package com.cashfree.sdk_sample.java;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.cashfree.sdk_sample.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void startDropCheckout(View view) {
        startActivity(new Intent(MainActivity.this, DropCheckoutActivity.class));
    }

    public void startElementCheckout(View view) {
        startActivity(new Intent(MainActivity.this, ElementCheckoutActivity.class));
    }

    public void startUPIIntentCheckout(View view) {
        startActivity(new Intent(MainActivity.this, UPIIntentActivity.class));
    }

    public void startWebCheckout(View view) {
        startActivity(new Intent(MainActivity.this, WebCheckoutActivity.class));
    }
}

File: app/src/main/java/com/cashfree/sdk_sample/kotlin/MainActivity.kt
package com.cashfree.sdk_sample.kotlin

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.cashfree.sdk_sample.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun startDropCheckout(view: View?) {
        startActivity(Intent(this@MainActivity, DropCheckoutActivity::class.java))
    }

    fun startElementCheckout(view: View?) {
        startActivity(Intent(this@MainActivity, ElementCheckoutActivity::class.java))
    }

    fun startUPIIntentCheckout(view: View?) {
        startActivity(Intent(this@MainActivity, UPIIntentActivity::class.java))
    }

    fun startWebCheckout(view: View?) {
        startActivity(Intent(this@MainActivity, WebCheckoutActivity::class.java))
    }

    fun startSubscriptionCheckout(view: View?) {
        startActivity(Intent(this@MainActivity, SubscriptionCheckoutActivity::class.java))
    }
}